website/docs/r/firewall_policy_application_rule_collection.html.markdown

```diff
@@ -0,0 +1,123 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_firewall_policy_application_rule_collection"
+description: |-
+  Manages a Firewall Policy Application Rule Collection.
+---
+
+# azurerm_firewall_policy_application_rule_collection
+
+Manages a Firewall Policy Application Rule Collection.
+
```

Comment:
Can we add a note here to clarify that mixing this resource and the `azurerm_firewall_policy_rule_collection_group`'s `application_rule_collection` property might cause problems?

Similar to: https://github.com/hashicorp/terraform-provider-azurerm/blob/bcee27c5951d43ff240d603965aba360bcf3ba0c/website/docs/r/subnet.html.markdown#L14-L16